src/pages/CleanestBuildings.vue

```diff
@@ -71,11 +72,11 @@ export default class CleanestBuildings extends Vue {
       The median building in our dataset emits {{ BuildingBenchmarkStats.GHGIntensity.median }}
       CO<sub>2</sub> kg / square foot, but these buildings are Chicago's best in class and emit
       <em>way</em> less! Some, like
-      <g-link to="/building/marina-towers">
+      <g-link to="/building-id/239096">
```

Comment:
I changed to using the building ID because this link broke, and IDs are more stable over time, so they are better for long-standing references.